code/modules/mob/living/carbon/human/species_types/plasmamen.dm

```diff
@@ -51,6 +51,14 @@
 
 /datum/species/plasmaman/spec_life(mob/living/carbon/human/H, delta_time, times_fired)
 	var/atmos_sealed = CanIgniteMob(H) && (isclothing(H.wear_suit) && H.wear_suit.clothing_flags & STOPSPRESSUREDAMAGE) && (isclothing(H.head) && H.head.clothing_flags & STOPSPRESSUREDAMAGE)
+	var/flammable_limb = FALSE
+	for(var/any_bodypart as anything in H.bodyparts)//If any plasma based limb is found the plasmaman will attempt to autoignite
+		var/obj/item/bodypart/found_bodypart = any_bodypart
```

Comment:
the point of `as anything` means you don't have to do var/any_bodypart and can just instead directly do `var/obj/item/bodypart/found_bodyparty as anything in H.bodyparts` because it won't typecheck.